packages/block-library/src/post-comments/edit.js

```diff
@@ -78,7 +44,138 @@ export default function PostCommentsEdit( {
+					) }
+				</Warning>
+
+				<h3>
+					{ __( 'One response to' ) } “{ postTitle }”
+				</h3>
+
+				<div className="navigation">
+					<div className="alignleft">
+						<a href="#top">« { __( 'Older Comments' ) }</a>
+					</div>
+					<div className="alignright">
+						<a href="#top">{ __( 'Newer Comments' ) } »</a>
+					</div>
+				</div>
+
+				<ol className="commentlist">
+					<li className="comment even thread-even depth-1">
+						<article className="comment-body">
+							<footer className="comment-meta">
+								<div className="comment-author vcard">
+									<img
+										alt=""
+										src="http://1.gravatar.com/avatar/d7a973c7dab26985da5f961be7b74480?s=32&d=mm&r=g"
+										className="avatar avatar-32 photo"
+										height="32"
+										width="32"
+										loading="lazy"
+									/>
+									<b className="fn">
+										<a href="#top" className="url">
+											{ __( 'A WordPress Commenter' ) }
+										</a>
+									</b>{ ' ' }
+									<span className="says">
+										{ __( 'says' ) }:
+									</span>
+								</div>
+
+								<div className="comment-metadata">
+									<a href="#top">
+										<time dateTime="2000-01-01T00:00:00+00:00">
+											{ __(
+												'January 1, 2000 at 00:00 am'
+											) }
+										</time>
+									</a>{ ' ' }
+									<span className="edit-link">
+										<a
+											className="comment-edit-link"
+											href="#top"
+										>
+											{ __( 'Edit' ) }
+										</a>
+									</span>
+								</div>
+							</footer>
+
+							<div className="comment-content">
+								<p>
+									{ __( 'Hi, this is a comment.' ) }
+									<br />
+									{ __(
+										'To get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.'
+									) }
+									<br />
+									{ __(
+										'Commenter avatars come from'
+									) }{ ' ' }
+									<a href="https://gravatar.com/">Gravatar</a>
+									.
+								</p>
+							</div>
+
+							<div className="reply">
+								<a
+									className="comment-reply-link"
+									href="#top"
+									aria-label="Reply to A WordPress Commenter"
+								>
+									{ __( 'Reply' ) }
+								</a>
+							</div>
+						</article>
+					</li>
+				</ol>
+
+				<div className="navigation">
+					<div className="alignleft">
+						<a href="#top">« { __( 'Older Comments' ) }</a>
+					</div>
+					<div className="alignright">
+						<a href="#top">{ __( 'Newer Comments' ) } »</a>
+					</div>
+				</div>
+
+				<div className="comment-respond">
+					<h3 className="comment-reply-title">
+						{ __( 'Leave a Reply' ) }
+					</h3>
+
+					<form className="comment-form" noValidate={ true }>
+						<p className="logged-in-as">
+							{ __( 'Logged in as' ) } { currentUserName }.{ ' ' }
+							<a href="#top">{ __( 'Log out?' ) }</a>{ ' ' }
+						</p>
```

Comment:
Not sure if we should remove this part here or if I should update https://github.com/WordPress/gutenberg/pull/40368 to include the "Logged in as {currentUser}". I was following https://github.com/WordPress/gutenberg/issues/26999#issuecomment-1099585096 in the overall design because if the user is not logged in the "Logged in as {currentUser}" line is not going to be displayed on the frontened.

Reply:
Sure. I'll remove it (the simplification proposed by @paaljoachim in https://github.com/WordPress/gutenberg/issues/26999#issuecomment-1099585096) to keep the consistency between the placeholders.

![image](https://user-images.githubusercontent.com/3305402/164388128-497baeb8-d9d5-4ebb-8047-1b203056b6ea.png)

Reply:
Done in https://github.com/WordPress/gutenberg/pull/40484/commits/fe62c6569b367ed2382772c863635362cc11114a.